tests/ci/rhcs_output/data.json

```diff
@@ -0,0 +1,4 @@
+{
+    "HtpasswdPass" : "1q2wFe4rpoe2318",
```

Comment:
This file is not supposed to be uploaded.

Reply:
ok, that's what I wanted to know before :) in that case I'll implement @amalykhi solution to generate random
chars for the password. we encountering github massage when pushing to the PR that this password is exposed 
and there is a leakage.